backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -84,6 +86,26 @@ public void testGetWaitingTime() throws Exception {
         assertEquals("Waiting Time per customer is", WAITING_TIME_MINS, waitingTime);
     }
 
+    @Test
+    public void testGetNewToken_success() throws Exception {
+        // Arrange.
+        CreateShopRequest shop = new CreateShopRequest(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
```

Comment:
Can we reuse addShopToRepository method for this?

Reply:
Done.